internal/graph/schema/alerts.graphqls

```diff
@@ -0,0 +1,197 @@
+extend type TeamEnvironment {
+	alerts(
+		"""
+		Get the first n items in the connection. This can be used in combination with the after parameter.
+		"""
+		first: Int
+
+		"""
+		Get items after this cursor.
+		"""
+		after: Cursor
+
+		"""
+		Get the last n items in the connection. This can be used in combination with the before parameter.
+		"""
+		last: Int
+
+		"""
+		Get items before this cursor.
+		"""
+		before: Cursor
+
+		"""
+		Ordering options for items returned from the connection.
+		"""
+		orderBy: AlertOrder
+
+		"""
+		Filter the returned objects
+		"""
+		filter: TeamAlertsFilter
+	): AlertConnection!
+}
+
+"""
+Alert interface.
+"""
+interface Alert implements Node {
+	"The unique identifier for the alert."
+	id: ID!
+	"The name of the alert."
+	name: String!
+
+	"The environment slug that the alert belongs to."
+	environmentName: String!
+
+	"The team slug that the alert belongs to."
+	teamSlug: Slug!
+
+	"The state of the alert."
+	state: AlertState!
+	"The query for the alert."
+	query: String!
+
+	"The duration for the alert."
+	duration: Float!
+
+	"The labels for the alert."
+	labels: [AlertKeyValue!]!
+
+	"The annotations for the alert."
+	annotations: [AlertKeyValue!]!
```

Comment:
Trenger vi å eksponere disse to? Føler det er litt ekstra implementasjon som vi heller kan vise som `workloads` og hva nå enn `annotations` brukes til?